### PR TITLE
wails: update sign transaction screen

### PR DESCRIPTION
**Pull request by maxime-carabina**
[duplicate of #326]

We are updating sign transaction screen to follow design : 

image

### CODE REVIEW

**Review by mazmassa — APPROVED** [review]

File: internal/handler/wallet/transfer.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func (t *transferCoin) Handle(params operations.TransferCoinParams) middleware.R
 
 	promptRequest := prompt.PromptRequest{
 		Action: walletapp.Transfer,
-		Msg:    fmt.Sprintf("Transfer %s nonaMassa from %s to %s, with fee %s nonaMassa", string(params.Body.Amount), wlt.Nickname, *params.Body.RecipientAddress, string(params.Body.Fee)),
+		Msg:    fmt.Sprintf("Transfer %s MAS from %s to %s, with fee %s MAS", string(params.Body.Amount), wlt.Nickname, *params.Body.RecipientAddress, string(params.Body.Fee)),
```
Comment (Thykof) [inline internal/handler/wallet/transfer.go:57]: Here you change the unit label without changing the value shown. Check if params.Body.Amount is really in MAS and not in nanoMAS (I believe it's nanoMas)

[optional]
I had in mind to pass here an object and format this message string in the frontend:
- in the backend we don't handle internationalization
- it's better to have all texts shown in the same place: en_US.json
I'll try to show you what I had in mind

Comment (Thykof) [reply internal/handler/wallet/transfer.go:57]: here is what I started: https://github.com/massalabs/thyra-plugin-wallet/pull/326/files but get stuck https://github.com/massalabs/massa-web3/issues/263

Comment (Thykof) [reply internal/handler/wallet/transfer.go:57]: you can cherry-pick commit https://github.com/massalabs/thyra-plugin-wallet/commit/a5483f93ab7b4a7160981d98271d70327897888f and solve potential conflicts

Comment (maxime-carabina) [reply internal/handler/wallet/transfer.go]: Done

**Review by Thykof — APPROVED** [review]

**Pull request merged**